### PR TITLE
update core to 1.141.2

### LINKED ISSUE #4014: Memory leak in yerpc
solution: update to a version of `@deltachat/jsonrpc-client` that has an updated version of `yerpc` that includes the fix https://github.com/deltachat/yerpc/pull/61

related: https://github.com/deltachat/yerpc/issues/60

This leads to dc desktop crashing for a user after some hours of usage: https://support.delta.chat/t/desktop-client-on-arm-mac-freezes-after-a-few-hours/3177

Basically response promises including their result objects are not garbage collected.

### PR DESCRIPTION
**Pull request by Simon-Laux**
closes #4014

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,9 @@
 - Update local help (2024-06-19)
 - refactor: safer types #3993
 - keep aspect ratio in quoted images #3999
+- Update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `1.141.2`
+  - fix memory leak in jsonrpc client code (to be specific in yerpc)
+  - support vcards exported by protonmail
```
Comment (r10s) [inline CHANGELOG.md:20]: i think, "Display vCard contact name in the message summary" is worth mentioning as well.
maybe also some fixes from >1.140.2

**Review by r10s — APPROVED** [review]

**Review by WofWca — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (iequidoo) [reply CHANGELOG.md:20]: Maybe "Case-insensitive search for non-ASCII messages" also

**Review by nicodh — APPROVED** [review]

**Review by iequidoo — APPROVED** [review]

**Review by farooqkz — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (Simon-Laux) [reply CHANGELOG.md:20]: since @farooqkz already merged this, I added those in a new pr in https://github.com/deltachat/deltachat-desktop/pull/4026

**Pull request merged**